assets/images/case-match.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 20 16" stroke="none" fillrule="evenodd">
```

Comment:
@jasonLaster these are the SVGs from Atom, what should we do in regards to license, etc? I think we should get the preamble from their license or something.